test/route.js

```diff
@@ -480,7 +500,7 @@ describe('Router', function () {
           .expect(200, cb)
         })
 
-        it('should work in a named parameter', function (done) {
+  /*      it('should work in a named parameter', function (done) {
```

Comment:
Just marking down the concern that there is a commented-out test in here still :)